packages/wasm/types/index.d.ts

```diff
@@ -0,0 +1,14 @@
+import { Plugin } from 'rollup';
+
+export interface RollupWasmOptions {
```

Comment:
Do we need this with the latest version of the typescript plugin, or will that do definition file output for us now?

Reply:
At the moment the types folder is a bit cleaner. I'm still refining the declaration output for the typescript plugin.